src/Routers/routes/UserRoutes.tsx

```diff
@@ -6,4 +7,11 @@ export default {
   "/users": () => <ManageUsers />,
   "/users/add": () => <UserAdd />,
   "/user/profile": () => <UserProfile />,
+  "/test": () => (
+    <RichTextEditor
+      initialMarkdown={""}
+      onChange={(value) => console.log(value)}
+      onAddNote={() => console.log("Add Note")}
+    />
+  ),
```

Comment:
I assume this'd be removed in the final version

Reply:
yes